addons/popochiu/engine/objects/gui/components/dialog_text/dialog_portrait/dialog_portrait.gd

```diff
@@ -5,6 +5,11 @@ extends PopochiuDialogText
 @onready var right_avatar_container: PanelContainer = %RightAvatarContainer
 @onready var right_avatar: TextureRect = %RightAvatar
 
+const _LOOKING_LEFT_DIRS := [
```

Comment:
Following [Godot's style guide](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html#constants-and-enums), constants should be defined without underscore on the first character.

```suggestion
const LOOKING_LEFT_DIRS := [
```